src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberContainerSymbol.cs

```diff
@@ -1320,7 +1320,7 @@ internal override bool HasPossibleWellKnownCloneMethod()
 
         internal override ImmutableArray<Symbol> GetSimpleNonTypeMembers(string name)
         {
-            if (_lazyMembersDictionary != null || declaration.MemberNames.Contains(name))
+            if (_lazyMembersDictionary != null || declaration.MemberNames.Contains(name) || declaration.Kind == DeclarationKind.Record)
```

Comment:
>declaration.Kind == DeclarationKind.Record [](start = 92, length = 42)

Is it possible to make this check more precise by actually checking if the ```name``` among those names that could be added for records?

Reply:
It's possible, but it would add complexity due to properties, their accessors and their backing fields. I don't think it's worthwhile.

---
In reply to: [567952525](https://github.com/dotnet/roslyn/pull/50900#discussion_r567952525) [](ancestors = 567952525)